inc/mailcollectorfeature.class.php

```diff
@@ -47,6 +47,7 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
     {
         switch ($item->getType()) {
             case PluginOauthimapApplication::class:
+                /** @var PluginOauthimapApplication $item */
```

Comment:
Switch with only one case is not optimized:
```suggestion
        if ($item instanceof PluginOauthimapApplication) {
```